src/types/tour.ts

```diff
@@ -10,7 +10,7 @@ export class Tour {
     @Type(() => Date)
     createdAt: Date
     @Type(() => Date)
-    updatedAt: Date
+    updatedAt: Date // todo: make nullable as for create this will not be set
```

Comment:
Are you sure? We'd have to check, but from .e.g Django, `updatedAt` has the same value as `createdAt` upon creation.

Reply:
Seems to be different from framework to framework. I only know it with updatedAt set to null when you create a new entry. But it actually doesn't really matter because we can find out either way if the data set is newly added or if it was updated. :)